content/neuvector-prime/azure/contents.lr

```diff
@@ -13,13 +13,23 @@ A step by step walk-through for creating a new offer from the Marketplace Overvi
 
 ## Basic (Project Details)
 
-1. Select existing subscription from the dropdown list. 
-2. Select existing Resource group from the dropdown list or create new. Provide name for new Resource group. 
+1. Select existing subscription from the dropdown list.
+2. Select existing Resource group from the dropdown list or create new Resource group.
+
+   Existing Resource Group selection:
+
+   ![existing resourcegroup](/images/existingresourcegroup.png "Existing ResourceGroup")
+
+   Create New Resource group:
+   Provide name for new Resource group.
+
+   ![new resourcegroup](/images/newresourcegroup.png "New ResourceGroup")
+
 3. Choose New cluster [ Yes/No ] option. 
    Yes option
    Choose Region
 
-   ![alt text](https://github.com/SUSE-Enceladus/marketplace-docs/tree/main/content/neuvector-prime/azure/images/basic.png?raw=true)
+   ![basic](/images/basic.png "basic")
```

Comment:
from what I've googled (and the docs for Lektor seem pretty terrible, maybe I haven't found the right one), this *looks* correct... but when I run it locally it doesn't load the images folder. If I move an image up to the same folder as the content and reference it without a path then it renders, but we dont want to dump a bunch of images there... trying to find a solution...

Reply:
ok, here's what worked for me:
move all of the images from `content/neuvector-prime/azure/images` to `assets/images/neuvector-prime/azure`
update the links to start with `/images/neuvector-prime/azure` 

I did this and everything renders for me, though the images are *huge* so I need to check whether markdown lets us set size constraints on images